File: doc/codes/csharp/examples/api/compute/pattern/uniform.cs
using System.Numerics;
using AUTD3;

namespace AUTD3.DocSamples.ApiComputePatternUniform;

internal static class Sample
{
    internal static void Run()
    {
        var geometry = new Geometry(new[] { new Device(Vector3.Zero) });

        var emission = new Emission(Phase.Zero, Intensity.Max);
        var dst = geometry.PatternBuffer();
        // ANCHOR: api
        Pattern.Uniform(emission, dst);
        // ANCHOR_END: api
    }
}

File: doc/codes/python/examples/api/compute/pattern/uniform.py
from autd3.geometry import Autd3, Geometry
from autd3.value import Emission, Intensity, Phase
from autd3_pattern import uniform

geometry = Geometry([Autd3([0.0, 0.0, 0.0], [1.0, 0.0, 0.0, 0.0])])

emission = Emission(
    Phase.ZERO,
    Intensity.MAX,
)
out = geometry.pattern_buffer()

# ANCHOR: api
uniform(emission, out)
# ANCHOR_END: api
